doc/recipe/preprocessor.rst

```diff
@@ -388,7 +391,11 @@ experiment is preferred for fx data retrieval:
       landmask:
         mask_landsea:
           mask_out: sea
-          fx_variables: [{'short_name': 'sftlf', 'exp': 'piControl'}, {'short_name': 'sftof', 'exp': 'piControl'}]
+          fx_variables: 
```

Comment:
I like the new syntax, but could you make it backward compatible so the old syntax keeps working, at least for a while?

Reply:
Maybe until version 2.4 or 2.5?

Reply:
I can try, but it won't look too good I think.

Reply:
yesh, I was asking a related question in my previous comment. I agree with @bouweandela on this one, when a new functional feature is implemented it's always nice to have a temporary overlap with its previous syntax :+1: